src/test/java/use_cases/add_edit_category_use_case/CategoryUCITest.java

```diff
@@ -41,7 +41,8 @@ void addCategoryInMonthSuccess() throws EntityException {
         // Check if the correct message is returned corresponding to the situation.
         assertEquals("You have added a new category!", interactor.addCategoryInMonth(addID1).getMessage());
         //Expected value is 2 because there is one default Category "Others" upon creation of each MonthlyStorage and one successful entry.
-        assertEquals(2, session.getMonthlyData(6).getCategoryData().size());
+        assertEquals(addID1.getName(), session.getMonthlyData(6).getCategoryData().get(1).getName();
+        assertEquals(addID1.getValue(), session.getMonthlyData(6).getCategoryData().get(1).getValue();
```

Comment:
Humm, why does it do that?

Reply:
Ah, I think it's because it's missing a bracket.

Reply:
Yeah, she is missing brackets at the end of both asserts.

Reply:
I'm not sure what you're talking about but I can fix the tests to what you commented

Reply:
Your tests are failing right now because of it, but once you add the brackets it should be fine.

Reply:
Actually, the second assertEquals will fail due to how I implemented the number casting methods. I think I'll add the first assertEquals and my original one if that's fine for you?

Reply:
Sure, if the name is there that means the Category has been added anyways so it should be fine!